### PR TITLE
display icons when mouse over on message

### PR DESCRIPTION
**Pull request by ruth4453**

### CODE REVIEW

**Review by ayeletBinder — APPROVED** [review]

File: client-side/src/stories/list/List.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { useState } from 'react';
```
Comment (e171271) [inline client-side/src/stories/list/List.jsx:2]: please join the 2 reacts imports

File: client-side/src/stories/list/List.jsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+import PropTypes from 'prop-types';
+import {
+    List,
+    ListItem,
+    ListItemText,
+    IconButton,
+} from '@mui/material';
+import CommentIcon from '@mui/icons-material/Comment';
+import { Button, Typography, Tooltip } from '@mui/material';
+import { Visibility, Delete, DataObject } from '@mui/icons-material';
```
Comment (e171271) [inline client-side/src/stories/list/List.jsx:13]: please join all mui imports to one

File: client-side/src/stories/list/List.jsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    };
+
+    const handleToggleRead =async (itemId,read,type) => {
+       const newMessage={
+        _id:itemId,
+        read:!read,
+        type:type
+       }
+       try {
+        await axios.put(`http://localhost:5000/message/${itemId}`,newMessage)
```
Comment (e171271) [inline client-side/src/stories/list/List.jsx:43]: move the basic url to variable - do it in all your code

File: client-side/src/stories/list/List.jsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+          .then(res => {
+            console.log(res.data)
```
Comment (e171271) [inline client-side/src/stories/list/List.jsx:45]: remove all coneole log

File: client-side/src/stories/list/List.jsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+          })
+      }
+      catch {
+        console.log("faild")
```
Comment (e171271) [inline client-side/src/stories/list/List.jsx:49]: console in catch should be coneole.error - do it in all your code

File: client-side/src/stories/list/List.jsx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+                        secondaryAction={
+                            <IconButton aria-label="comment">
+                                <CommentIcon />
+                            </IconButton>
+                        }
+                    >
+                        <ListItemText primary={type.type} className={read ? 'read' : 'unread'} />
+                        {hoveredItem === _id && (
+                            <Tooltip title={read ? 'Mark as unread' : 'Mark as read'}>
+                                <div className="icons-wrapper">
```
Comment (e171271) [inline client-side/src/stories/list/List.jsx:84]: move the titles to constants files

File: client-side/src/stories/list/List.jsx [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+                                    <IconButton
+                                        className="eye-icon"
+                                        onClick={() => handleToggleRead(_id,read,type)}
+                                        style={{ color: read ? '#474747' : 'rgb(27, 27, 27)' }}
+                                    >
```
Comment (e171271) [inline client-side/src/stories/list/List.jsx:90]: styles should be in scss file

File: client-side/src/stories/list/List.jsx [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+    ListItemText,
+    IconButton,
+} from '@mui/material';
+import CommentIcon from '@mui/icons-material/Comment';
+import { Button, Typography, Tooltip } from '@mui/material';
+import { Visibility, Delete, DataObject } from '@mui/icons-material';
+import './List.scss';
+
+
+const ListComponent = ({dataObject}) => {
```
Comment (e171271) [inline client-side/src/stories/list/List.jsx:17]: please add default value

File: client-side/src/stories/list/List.scss [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+
+
+.wrapper {
+
+    background-color: #ffffff;
+    padding: 12px;
+    position: absolute;
+    right: 10px;
+    top: 50px;
+    width: 220px;
```
Comment (e171271) [inline client-side/src/stories/list/List.scss:11]: please use % instead of px
and move all colors to variables
do it in all the file

**Review by e171271 — CHANGES_REQUESTED** [review]

**Review by SariGlick — APPROVED** [review]

**Review by e171271 — APPROVED** [review]

**Pull request merged**